db/migrate/20241224142141_add_total_paid_amount_cents_to_invoices.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class AddTotalPaidAmountCentsToInvoices < ActiveRecord::Migration[7.1]
+  disable_ddl_transaction!
+
+  def up
+    add_column :invoices, :total_paid_amount_cents, :bigint, null: true, default: 0
```

Comment:
will in the future we update the existing paid invoices with real values? 🤔

Reply:
Yes I will do a migration to fill the values i was thinking on another pr but can be on this one

Reply:
I need to do the migration later, not on this PR, because I will sync some invoices. However, new ones will have a value of 0 and will only be properly updated when we change the code later. So, when we change the code, we should perform the migration at the same time. For now, just adding the field is fine; we won’t use it yet.

this way I don't have to migrate the data 2 times